src/app/data.service.ts

```diff
@@ -1,164 +1,57 @@
 import {Injectable} from '@angular/core';
-import {Area} from './model/Area';
+import {LeagueTablePosition} from './model/LeagueTablePosition';
+import {Observable} from 'rxjs';
+import {HttpClient} from '@angular/common/http';
+import {environment} from '../environments/environment';
+import {map} from 'rxjs/operators';
 import {Team} from './model/Team';
-import {TableInput} from './model/TableInput';
-import {Standing} from './model/Standing';
-import {Competition} from './model/Competition';
-import {Season} from './model/Season';
-import {StandingInput} from './model/StandingInput';
-import {Observable, of} from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
 export class DataService {
-  // private areas: Array<Area>;
-  // private teams: Array<Team>;
-  // private tableInputs: Array<TableInput>;
-  standingInput: StandingInput;
 
-  getStandingInput(): Observable<StandingInput> {
-    return of(this.standingInput);
+/*  getLeagueAllTableResults(competitionId: number): Observable<Array<LeagueTablePosition>>{
+    return this.http.get<Array<LeagueTablePosition>>(environment.restUrl + '/results/getLeagueResults/' + competitionId)
+      .pipe(
+        map(data => {
+          const leagueTablePositionList = new Array<LeagueTablePosition>();
+          for (const leagueTablePosition of data){
+            leagueTablePositionList.push(LeagueTablePosition.fromHttp(leagueTablePosition));
+          }
+          return leagueTablePositionList;
+        })
+      );
+  }*/
+
+  getLeagueTableResultsOfType(competitionId: number, type: string): Observable<Array<LeagueTablePosition>>{
+    return this.http.get<Array<LeagueTablePosition>>('/api/results/league-results?league_id=' + competitionId
```

Comment:
Nie używaj konkatenacji - JavaScript ma String template https://hacks.mozilla.org/2015/05/es6-in-depth-template-strings-2/